app/view_models/verify/base.rb

```diff
@@ -36,6 +36,18 @@ def message
       html_paragraph(text: I18n.t("idv.modal.#{step_name}.#{error}")) if error
     end
 
+    def unsupported_jurisdiction_error(sp_name)
```

Comment:
Why include this in the base class and not the `SessionNew` view model?

Reply:
Honestly, I think when we were working through it together we stuck it here, and I never really thought about moving it? What would be the advantage of moving it into the view model?

Reply:
It's more clear that this method is specific to the sessions view and is not used by the phone view.